index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <link rel="stylesheet" href="style.css">
+
+    <div class="container">
+        <div class="header-nav">
+            <p class="MyLogo">My Logo</p>
+            <div class="topNav">
+                <a href="#Portfolio">Portfolio</a>
+                <a href="#About">About</a>
+                <a href="#Contact">Contact</a>
+            </div>
+        </div>
+        <div class="box2">
+            <h1>I'm Henry<br>Glad to see you!</h1>
+         <p class="Content">
+            I'm a software developer! I can help you build a product ,
+                    feature or website Look through<br>some of my work and 
+                    experience! If you like what you see and have a project
+                    you need<br>coded, don't hestiate to contact me.
+         </p>
+
+         <p class="contact">LET'S CONNECT</p>
+         <img src="imgs/social media icons.png">
```

Comment:
[ ] kindly change this `img` tag, it should be a list of icons, use a `ul` tag to group them, and `li` per icon; each icon has a link to your social media accounts. 👀